src/cargo/core/shell.rs

```diff
@@ -406,16 +406,16 @@ impl Shell {
         Ok(())
     }
 
-    /// Prints the passed in [Message] to stderr
-    pub fn print_message(&mut self, message: Message<'_>) -> std::io::Result<()> {
+    /// Prints the passed in &[[`Group`]] to stderr
+    pub fn print_report(&mut self, groups: &[Group<'_>]) -> std::io::Result<()> {
```

Comment:
Should we refer to `Report` type and `reports` instead of `&[Grouo]` / `groups`?